docs/antrea-ipam.md

```diff
@@ -218,6 +218,34 @@ metadata:
 A StatefulSet Pod's IP will be kept after Pod restarts, when the IP is allocated from the
 annotated IPPool.
 
+### IPAM behaviors
+
+When `AntreaIPAM` is enabled, `antrea-controller` will watch IPPool CRs and StatefulSets
+from K8s apiserver, and `antrea-agent` will register IPAM driver to CNI server.
+
+#### On IPPool CR create/update event
+
+`antrea-controller` will update IPPool counters, and periodically clean up stale IP addresses.
+
+#### On StatefulSet create event
+
+`antrea-controller` will check annotations and preallocate IPs from the IPPool.
+
+#### On StatefulSet delete event
+
+`antrea-controller` will clean up IP allocations under this StatefulSet.
+
+#### On Pod create
+
+`antrea-agent` will receive a CNI add request, it will check annotations and allocate an IP from
+an IPPool. The IP is a pre-allocated StatefulSet IP, a specified IP or the next available IP in
```

Comment:
"the corresponding IPPool"

Or, probably we should remove "from an IPPool", given you have another sentence to explain the IP, which seems can be pre-allocated too.

Consider "and allocate an IP for the Pod, which can be a pre-allocated IP StatefulSet IP, a user-specified IP, or the next available IP in the specified IPPool."

Reply:
The IP can be

Reply:
Refactored this section to
```
`antrea-agent` will receive a CNI add request, and it will check the Antrea IPAM annotations and
allocate an IP for the Pod, which can be a pre-allocated IP StatefulSet IP, a user-specified IP,
or the next available IP in the specified IPPool.
```